### DISCUSSION TITLE
Animal TDM

### DISCUSSION THREAD

**Discussion by cookievet** [LAPKB/Pmetrics#168] [Q&A]
Hi all,
Would anyone be interested in starting some animal studies using PK data and TDM?
I am a UK veterinary surgeon working with cats and anti-virals and really need to be able to use TDM to optimise the doses of these expensive meds.
Any and all help would be much appreciated.
Kind regards, Steve
Dr Stephen W. Cooke BVSc, MRCVS.
Oxford, UK

**Comment by mnneely** [comment]
What did you have in mind? I published one study years ago with Ronette Gehring on tulathromycin PK in tortoises, so vet medicine is not completely foreign… Of course, being a pediatrician, I’ve been compared to veterinarians in the past, but I don’t actually work with animals. :)

**Comment by cookievet** [comment]
Many thanks for the swift reply.I am monitoring plasma GS-441524 in cats being treated for a feline coronavirus disease called Feline Infectious Peritonitis. The known PK data is sparse, 5 x healthy cats but we (Edinburgh Veterinary School and other universities) are seeing the need for TDM in order to maintain plasma levels in the 10 microM range when cats are responding in very different ways to similar doses. The main problem is that the meds are expensive so the TDM is necessary to ensure the client’s funds are spent wisely.I am trying to find a way of entering the healthy cat data into software so I can try to fit models from ill cats to their plasma values at various doses.Does that help?I will have a read of your paper, it sounds fascinating and I am a ‘small and exotic animal’ vet, similar to little people!SteveDr Stephen W. Cooke BVSc, MRCVSOn 11 May 2023, at 17:16, Michael Neely ***@***.***> wrote:﻿
What did you have in mind? I published one study years ago with Ronette Gehring on tulathromycin PK in tortoises, so vet medicine is not completely foreign… Of course, being a pediatrician, I’ve been compared to veterinarians in the past, but I don’t actually work with animals. :)

**Comment by mnneely** [comment]
Sounds really interesting. Any chance you'd be able to attend our upcoming Pmetrics workshop in Lyon, France July 6-7? We are focusing on antibiotic modeling and dose calculations based on probability of target attainment. Would be perfect for you and we could meet in person. Details on our website: www.lapk.org. We can be species agnostic!
